Quiz/src/jsx/components/Answer.jsx

```diff
@@ -1,32 +1,30 @@
 import React from "react";
 
-class Answer extends React.Component{
-    constructor(props) {
-        super(props);
+const Answer = props => (
```

Comment:
Where possible, functional components are IMO better because you get an immediate assurance that there is no lifecycle and no state, which makes it easier to reason about contents. 

Un-bracketed arrow functions are even better because they are 'point-free', ie there are no internal references, just a return expression. Previously there were 2 points: declare a variable to define the `options`, then return a block of content which references the `options`. I prefer this because it's less moving parts to remember – some people dislike it on the grounds it encourages that any logic (ie the `options.map`) gets put in the content (I don't see a problem with that 🤷‍♂️ ).